src/main/java/crawlercommons/utils/Strings.java

```diff
@@ -0,0 +1,28 @@
+package crawlercommons.utils;
+
+/**
+ * Util functions for manipulating strings.
+ */
+public class Strings {
+
+    public static boolean isBlank(final String cs) {
+        if (cs == null || cs.isEmpty()) {
```

Comment:
Change cs.isEmpty()
to
cs.trim().isEmpty()

And you will be able to cut several lines from the code

Reply:
Can you clarify what lines exactly could be cut? The method `trim()` might unnecessarily allocate and copy bytes into a new String object. Also, I'm not sure if `trim()` removes the same chars as `Character.isWhitespace()`. This implementation is similar to the StringUtils class used in apache `http-components` which used in their URL parsing code.

Reply:
Can you clarify what lines exactly could be cut? The method `trim()` might unnecessarily allocate and copy bytes into a new String object. Also, I'm not sure if `trim()` removes the same chars as `Character.isWhitespace()`. This implementation is similar to the `StringUtils` class used in Apache's http-components library URL parsing code.